blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -59,197 +70,279 @@ blueprint:
       default: []
       selector:
         action:
-    # inputs for looping custom actions on long button press events until the corresponding release event is received
     button_long_loop:
```

```suggestion
    # inputs for looping custom actions on long button press events until the corresponding release event is received
    button_long_loop:
```

keep comment since it improves code readability